src/lib/components/Card/Card.tsx

```diff
@@ -22,29 +23,40 @@ export interface FlowbiteCardImageTheme {
   horizontal: FlowbiteBoolean;
 }
 
-export interface CardProps extends PropsWithChildren<ComponentProps<'div'>> {
+type CommonCardProps = PropsWithChildren<ComponentProps<'div'>> & {
   horizontal?: boolean;
   href?: string;
-  imgAlt?: string;
-  imgSrc?: string;
   theme?: DeepPartial<FlowbiteCardTheme>;
-}
+};
+
+export type CardProps =
```

Comment:
I am not sure if it is possible to create XOR types with "interfaces". If you feel strongly about the usage of "type" instead of "interface" I can try to improve this. However I then also recommend adding a linter rule which disallows the usage of "type".

Reply:
I would always prefer to use `Interface`. Can you change it?